data-raw/USGS_CAWSC.R

```diff
@@ -233,13 +239,19 @@ USGS_CAWSC <- cawsc_sign_c %>%
       CharacteristicName == "Chlorophyll a" ~ "Chlorophyll",
       CharacteristicName == "Inorganic nitrogen (nitrate and nitrite)" ~ "DissNitrateNitrite",
       CharacteristicName == "Organic carbon" ~ "DOC",
-      CharacteristicName == "Orthophosphate" ~ "DissOrthophos"
+      CharacteristicName == "Orthophosphate" ~ "DissOrthophos",
+      CharacteristicName == "Oxygen" ~ "DissolvedOxygen",
+      CharacteristicName == "pH" ~ "pH",
+      CharacteristicName == "Temperature, water" ~ "Temperature",
+      CharacteristicName == "Specific conductance" ~ "Conductivity"
     )
   ) %>%
   pivot_wider(
     names_from = CharacteristicName,
     values_from = c(Value, Sign),
-    names_glue = "{CharacteristicName}_{.value}"
+    names_glue = "{CharacteristicName}_{.value}",
+    #remove duplicates - 2 - DO, 13 - pH, 4 - Temp., 14 - SpC
+    values_fn = "first"
```

Comment:
Do you have any idea why there were duplicates?

Reply:
@sbashevkin There are duplicates of 'historical' and more recently collected samples. Historical samples typically don't have a time associated with them and the row with a duplicate date has a unique value. The more recently collected samples with duplicates have identical values and I'm not sure why they exist in the database.